packages/arb-token-bridge-ui/src/hooks/useChainLayers.ts

```diff
@@ -15,11 +15,17 @@ function getChainLayerByChainId(chainId: number): ChainLayer {
   return 'Orbit'
 }
 
-export const useChainLayers = () => {
+export const useChainLayers = ({
+  parentChainIdOverride,
+  childChainIdOverride
+}: {
+  parentChainIdOverride?: ChainId
```

Comment:
We need to get layers for specific transactions, so we need to get data for chain IDs different than the selected networks.